src/ngraph/runtime/gpu/gpu_external_function.hpp

```diff
@@ -80,7 +80,7 @@ namespace ngraph
                 EntryPoint m_compiled_function;
 
             private:
-                void collect_unique_functions();
+                void optimize_and_assemble();
```

Comment:
nid: a more meaningful name than `optimize`

Reply:
Oops, that function's definition is removed but not it's declaration. I'll fix that. Thanks

Reply:
I see.